src/org/opensolaris/opengrok/history/HistoryGuru.java

```diff
@@ -341,64 +341,67 @@ public void addRepositories(File[] files, Collection<RepositoryInfo> repos,
      */
     private void addRepositories(File[] files, Collection<RepositoryInfo> repos,
             IgnoredNames ignoredNames, boolean recursiveSearch, int depth) {
+
         for (File file : files) {
-            Repository repository = null;
```

Comment:
yep, mixing two things, very hard to review and retest

Reply:
but it looks like all accurev changes come from #1721

Reply:
Yes, those changes are from that issue. So I must not be doing the branching correctly.  To start, I first made a fork from the main OpenGrok github repository.  I then made the AccuRev changes and pushed them up to my repository, then made a pull request. I then made a branch from my repository to deal with the new PowerShell code. So I guess its going to have the Accurev changes in it. How else would I do the branching to keep things separate?

Reply:
Note, I am not using command line git commands, but rather the web interface provided.

Reply:
:) you can always create clean branches, but you need to go to git cli, git UIs are a bit oversimplistic and sometimes hide the fantastic possibilities that one can do with git ...